gradle/libs.versions.toml

```diff
@@ -90,6 +91,7 @@ asciidoctorj-pdf = { module = "org.asciidoctor:asciidoctorj-pdf", version.ref =
 awsS3 = { module = "software.amazon.awssdk:s3", version.ref = "s3" }
 clikt = { module = "com.github.ajalt.clikt:clikt", version.ref = "clikt" }
 commonsCompress = { module = "org.apache.commons:commons-compress", version.ref = "commonsCompress" }
+commonsText = { module = "org.apache.commons:commons-text", version.ref = "commonsText" }
```

Comment:
I'm not super happy about introducing a dependency on `commons-text` just for escaping XML... can we find a more lightweight solution?

Reply:
Fell free to suggest one!

I didn't find anything by googling. The Cyclone DX lib delegates to Jackson for the XML generation that delegates to StAX, but I found nothing there too :disappointed:

Reply:
Also, I remember that somewhere we had a discussion about wrapping Copyright texts in `CDATA`, but I currently cannot find it anymore. Would that also solve the issue?

Reply:
Somewhat related, though targeting the description: https://github.com/oss-review-toolkit/ort/issues/2814

Reply:
> Also, I remember that somewhere we had a discussion about wrapping Copyright texts in `<![CDATA[...]]>`, but I currently cannot find it anymore. Would that also solve the issue?

Assuming that this would help, I'm wondering why our Copyright's aren't already wrapped in `<![CDATA[...]]>`. Also this [test from upstream](https://github.com/CycloneDX/cyclonedx-core-java/blob/6cfef328eb0c6c73f83dff13f9b2fe1934c271b9/src/test/resources/1.5/valid-evidence-1.5.xml#L27) shows that the text is nested, which does not seem to be the case for our reports. So maybe that's the root cause: For some reason we're not using the `<text>` tag, and that's also why data does not get wrapped in `<![CDATA[...]]>`. Could you investigate into that direction, @nnobelis?

Reply:
In the test you linked, the copyright is under an `evidence` element:
```
<component type="application">
            <evidence>
                <copyright>
```

And `Component.evidence.copyright` has a special class in the model, with this `text` element:
https://github.com/CycloneDX/cyclonedx-core-java/blob/6cfef328eb0c6c73f83dff13f9b2fe1934c271b9/src/main/java/org/cyclonedx/model/Copyright.java#L30

If you use `Component.copyright` instead, this is just a string:
https://github.com/CycloneDX/cyclonedx-core-java/blob/6cfef328eb0c6c73f83dff13f9b2fe1934c271b9/src/main/java/org/cyclonedx/model/Component.java#L174

Reply:
Agreed, it seem odd that Copyrights are wrapped in `CDATA` if part of `<evidence>`, but not otherwise.